app/src/main/java/org/covidwatch/android/presentation/HomeFragment.kt

```diff
@@ -6,16 +6,25 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
 import androidx.navigation.fragment.findNavController
 import org.covidwatch.android.BuildConfig
 import org.covidwatch.android.R
 import org.covidwatch.android.databinding.FragmentHomeBinding
+import org.covidwatch.android.domain.FirstTimeUser
+import org.covidwatch.android.domain.ReturnUser
+import org.covidwatch.android.domain.Setup
+import org.covidwatch.android.domain.UserFlow
+import org.covidwatch.android.presentation.home.HomeViewModel
+import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class HomeFragment : Fragment(R.layout.fragment_home) {
 
     private var _binding: FragmentHomeBinding? = null
     private val binding get() = _binding!!
 
+    private val homeViewModel: HomeViewModel by viewModel()
```

Comment:
Do you need to pass the repo? Or does Koin help with this?

Reply:
koin does it for us :)